src/senaite/core/profiles/default/workflows/senaite_sample_workflow/definition.xml

```diff
@@ -413,6 +415,7 @@
     <exit-transition transition_id="receive"/>
     <exit-transition transition_id="submit"/>
     <exit-transition transition_id="reject" />
+    <exit-transition transition_id="listing_multi_results"/>
```

Comment:
I think we need a guard for this transition to ensure this transition is possible for samples in `sample_due` status only if there are "field" analyses (analyses meant to be registered at point of collection) that are not yet submitted. If the status of the sample is `received`, the guard would return True if there is at least one analysis (from "lab") not yet submitted.